### DISCUSSION TITLE
Is it possible to scale locally a panels / components?

### DISCUSSION THREAD

**Discussion by zenoxs** [emilk/egui#4141] [Q&A]
Hello,

Firstly, I'd like to express my gratitude for the egui library; it has been delightful to use.

I'm currently utilizing egui to develop the UI for my game, specifically leveraging the `egui_settings.scale_factor` to adjust the UI scale in response to window resolution changes. However, I've encountered a challenge: I also use egui for debug tool panels, but I do not want these to scale with the resolution as they become excessively large at my target resolution. Is there a way to selectively scale certain UI panels/components without applying a global scale to all of egui's elements?

Thank you!

**Comment by YgorSouza** [comment]
This was implemented in #3906 but has not been released yet. See the Pan Zoom example in the demo app.

https://github.com/emilk/egui/blob/a1d5145c16aba4d0b11668d496735d07520d0339/crates/egui_demo_lib/src/demo/pan_zoom.rs
